website/src/components/dbt-editor/tailwindStyles.css

```diff
@@ -0,0 +1,7 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
We shouldn't need to include tailwind long term if we improve how we bundle this repo